<launch>
	
  	<!-- JOY ROS NODE -->
	<node respawn="true" pkg="joy" type="joy_node" name="ps4_joy" >
    	<param name="dev" type="string" value="/dev/input/js1" />
    	<param name="deadzone" value="0.12" />
 	</node>
       
        <node name="joy_control" pkg="nano_drone" type="joy_control.py" output="screen" />

</launch>
